wave-fe/src/app/clues/clues.component.ts

```diff
@@ -1,13 +1,33 @@
 import { Component, OnInit, Input } from '@angular/core';
+import { Observable } from 'rxjs';
+import { GuessService } from '../guess.service';
+import { RoundService } from '../round.service';
 
 @Component({
   selector: 'app-clues',
   templateUrl: './clues.component.html'
 })
 export class CluesComponent implements OnInit {
-
-  constructor() { }
   @Input() clues: any;
+  isGuestPlayer$: Observable<boolean>
+
+  constructor(private guessService: GuessService, private roundService: RoundService) {
+    this.isGuestPlayer$ = roundService.isGuestPlayer$;
+  }
+
+  get startClasses() {
+    const color = this.clues?.startColor;
+    return `bg-${color}-400 text-${color}-400`;
+  }
+
+  get endClasses() {
```

Comment:
Should it be `get endClasses(): string {` @diego0020 ?

Reply:
yes

Reply:
Updated